appserver/web/gf-web-connector/src/main/java/fish/payara/appserver/context/JavaEEContextUtilImpl.java

```diff
@@ -85,6 +82,15 @@ void init() {
         doSetInstanceContext();
     }
 
+    public ServerContext getServerContext() {
```

Comment:
Just to make sure we make no changes, this should be package private.

Reply:
FYI: This was not a miss. When the changes were still in separate branches making it this method `protected` would result in an compile error in my IDE for some odd reason. That is why I left it `public`. Now with everything in one branch it behaves as expected and lets me change it without complaining.